eras/conway/impl/testlib/Test/Cardano/Ledger/Conway/Imp.hs

```diff
@@ -58,11 +61,21 @@ spec ::
   Spec
 spec = do
   BabbageImp.spec @era
-  describe "ConwayImpSpec" $ withImpState @era $ do
-    Enact.spec @era
-    Epoch.spec @era
-    Gov.spec @era
-    GovCert.spec @era
-    Utxo.spec @era
-    Utxos.spec @era
-    Ratify.spec @era
+  describe "ConwayImpSpec - post bootstrap (protocol version 10)" $
+    withImpStateWithProtVer @era (natVersion @10) $ do
+      Enact.spec @era
+      Epoch.spec @era
+      Gov.spec @era
+      GovCert.spec @era
+      Utxo.spec @era
+      Utxos.spec @era
+      Ratify.spec @era
+  describe "ConwayImpSpec - bootstrap phase (protocol version 9)" $
+    withImpState @era $ do
+      Enact.relevantDuringBootstrapSpec @era
+      Epoch.relevantDuringBootstrapSpec @era
+      Gov.relevantDuringBootstrapSpec @era
+      GovCert.relevantDuringBootstrapSpec @era
+      Utxos.relevantDuringBootstrapSpec @era
+      Utxo.spec @era
```

Comment:
I am probably being too pedantic, but could you please order these in the same as the non-bootstrap specs:
```suggestion
      Utxo.spec @era
      Utxos.relevantDuringBootstrapSpec @era
```

Reply:
100%, it's bugging me too now that you pointed it out. Done